components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -3256,6 +3264,14 @@ func TestDocuments_ValidateEvent(t *testing.T) {
 				doc := fixORDDocument()
 				doc.EventResources[0].PartOfConsumptionBundles[0].DefaultTargetURL = "https://exmaple.com/test/v3"
 
+				return []*open_resource_discovery.Document{doc}
+			},
+		}, {
+			Name: "Empty `PartOfConsumptionBundle` field for Event",
```

Comment:
Missing test for: `an API without targetURLs and not part of consumption bundle should be valid`